src/components/app/SingleStrategy/index.tsx

```diff
@@ -76,12 +86,22 @@ export const SingleStrategy = () => {
             try {
                 const loadedStrategy = await getStrategies([strategyId]);
                 setStrategyData(loadedStrategy);
+                console.log(loadedStrategy);
                 setIsLoading(false);
             } catch (error) {
                 console.log('Error:', error);
                 setIsLoading(false);
                 setError(error);
             }
+            try {
+                const loadedVault = await getVault(vaultId);
+                setVault(loadedVault);
+                setIsVaultLoading(false);
+            } catch (error) {
+                console.log('Error:', error);
+                setIsVaultLoading(false);
+                setError(error);
+            }
         };
         loadStrategyData();
     }, [strategyId]);
```

Comment:
if we add the `vaultId` to inputs to useEffect does it work?

Reply:
I added it and the strategy page still loads, but I don't know how to test if it works.